doc/sfwbar.rst

```diff
@@ -707,6 +707,30 @@ tricky). Mapping is supported via top-level ``MapAppId`` keyword. I.e. ::
 where app_id is the desired app_id and pattern is a regular expression to
 match the title against.
 
+If you are using an XWayland app, they usually do not have an `app_id` set. If
+an icon is not showing, you can add your icon to the following locations:
+1. `$HOME/.icons`
+2. One of the directories listed in `$XDG_DATA_DIRS/icons`
+3. /usr/share/pixmaps
+
+If an `app_id` is not set, sfwbar will fallback to using the `instance` in the
+`window-properties`, which can be found by using a tool such as `swaymsg` (if on sway)
```

Comment:
We should probably say that users can check programs app_id with Hyprland using 'hyprctl -j clients' and for other compositors supporting wlr-foreign-toplevel protocol (i.e. labwc, wayfire) using 'sfwbar -d -g app_id`

Reply:
I also just made a change in the latest git version to support 'sfwbar -d -g app_id' with all IPCs. This should make icon lookup easier across different compositors.